server/features/planning.feature

```diff
@@ -50,7 +50,7 @@ Feature: Planning
         """
             {"_items": [{
                 "guid": "__any_value__",
-                "type": "planning",
```

Comment:
`type` is actually required, as it is used by the publishing system (`type` is a field available through all content resources, and so we're using it from Planning based resources too).

`_type`: Resource Type
`type`: Item Type - In content authoring this is the sub-type, such as `text`, `picture`, `video` etc